disdrodb/L0/retrive_reader.py

```diff
@@ -0,0 +1,96 @@
+import os
+import sys
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_all_readers() -> dict:

```

Comment:
I suggest to name it: `get_available_readers`

Reply:
ok